html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label>
+      <br />
+      <input type="email" id="email" name="email" />
+      <br />
+      <label for="select">Why is coding cool?</label>
+      <br />
+      <select id="cool" name="'Why is coding cool?">
+        <option>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
+      </select>
+      <br />
+      <br />
+      <label for="radio">Favorite Language (choose one)</label>
+      <br />
+      <br />
+      <input type="radio" id="html" name="language" />
+      <label for="html">HTML</label>
+      <input type="radio" id="css" name="language" />
+      <label for="css">CSS</label>
+      <input type="radio" id="javascript" name="language" />
+      <label for="javascript">Javascript</label>
+      <br />
+      <br />
+      <label for="checkbox"
+        >Would you like to sign up for more information?</label
+      >
```

Comment:
Since this `label` isn't associated with any `input`, this should just be a `p` instead